model/_global/search/structures.smithy

```diff
@@ -0,0 +1,622 @@
+    default_operators: DefaultOperator,
+
+    @httpQuery("df")
+    df: String,
+
+    @httpQuery("docvalue_fields")
+    docvalue_fields: String,
+
+    @httpQuery("expand_wildcards")
+    expand_wildcards: ExpandWildcards,
+
+    @httpQuery("explain")
+    explain: Boolean,
+
+    @httpQuery("from")
+    from: Integer,
+
+    @httpQuery("ignore_throttled")
+    ignore_throttled: Boolean,
+
+    @httpQuery("ignore_unavailable")
+    ignore_unavailable: Boolean,
+
+    @httpQuery("lenient")
+    lenient: Boolean,
+
+    @httpQuery("max_concurrent_shard_requests")
+    max_concurrent_shard_requests: Long,
+
+    @httpQuery("pre_filter_shard_size")
+    pre_filter_shard_size: Long,
+
+    @httpQuery("preference")
+    preference: String,
+
+    @httpQuery("q")
+    q: String,
+
+    @httpQuery("request_cache")
+    request_cache: Boolean,
+
+    @httpQuery("rest_total_hits_as_int")
+    rest_total_hits_as_int: Boolean,
+
+    @httpQuery("routing")
+    routing: String,
+
+    @httpQuery("scroll")
+    scroll: Time,
+
+    @httpQuery("search_type")
+    search_type: SearchType,
+
+    @httpQuery("seq_no_primary_term")
+    seq_no_primary_term: Boolean,
+
+    @httpQuery("size")
+    size: Integer,
+
+    @httpQuery("sort")
+    sort: UserDefinedValueList,
+
+    @httpQuery("source")
+    source: String,    
+
+    @httpQuery("source_excludes")
+    source_excludes: UserDefinedValueList,
+
+    @httpQuery("source_includes")
+    source_includes: UserDefinedValueList,
+
+    @httpQuery("stats")
+    stats: String,
+
+    @httpQuery("stored_fields")
+    stored_fields: Boolean,
+
+    @httpQuery("suggest_field")
+    suggest_field: String,
+
+    @httpQuery("suggest_mode")
+    suggest_mode: SuggestMode,
+
+    @httpQuery("suggest_size")
+    suggest_size: Long,
+
+    @httpQuery("suggest_text")
+    suggest_text: String,
+
+    @httpQuery("terminate_after")
+    terminate_after: Integer,
+
+    @httpQuery("timeout")
+    timeout: Time,  
+
+    @httpQuery("track_scores")
+    track_scores: Boolean,
+
+    @httpQuery("track_total_hits")
+    track_total_hits: Integer,
+
+    @httpQuery("typed_keys")
+    typed_keys: Boolean,
+
+    @httpQuery("version")
+    version: Boolean,
+    // Common options end
+
+    // Request-body parameters 
+
+    ///////////////////////////////////////////////////
+    //                                               //
+    //   QueryPARAMETERS that are in-common          //
+    //   with REQUEST-BODY are mentioned             //
+    //   as comment.                                 //
+    //                                               //
+    ///////////////////////////////////////////////////
+
+    // docvalue_fields: String,
+    // explain: Boolean,
+    // from: Integer,
+    // seq_no_primary_term: Boolean,
+    // size: Integer,
+    // source: String,    
+    // stats: String,
+    // terminate_after: Integer,
+    // timeout: Time, 
+    // version: Boolean,              
```

Comment:
There is a workaround to include both:

{
    @httpQuery("size")
    size_query_parameter: Integer,
    size: Integer,
}

Reply:
Done